recipes-core/initrdscripts/tegra-flash-init_1.0.bb

```diff
@@ -25,5 +28,5 @@ do_install() {
 }
 
 FILES:${PN} = "/"
-RDEPENDS:${PN} = "util-linux-blkdiscard tegra-flash-reboot mtd-utils e2fsprogs-mke2fs libusbgx-tegra-initrd-flash watchdog-keepalive gptfdisk"
+RDEPENDS:${PN} = "util-linux-blkdiscard tegra-flash-reboot mtd-utils e2fsprogs-mke2fs libusbgx-tegra-initrd-flash watchdog-keepalive gptfdisk bash"
```

Comment:
See above.  It would be preferable to keep bash out of the dependencies here.

Reply:
Removed the bash dependency.

Reply:
Removed the bash dependency.